templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+
+<%- dirnames.each do |dir| -%>
+  Copy-Item <%= "#{dir.sub("/cygdrive/c", "C:/ProgramData/chocolatey/lib/#{project.name}")}/*" %> <%= dir.sub('/cygdirve/c', 'C:') %>
```

Comment:
@ferventcoder this is where I am now. The files are getting installed into the chocolatey default directory, and I want to copy them from there to the specified location. I'm in the process of testing this at the moment, though, so I'll likely have more updates